test/Ocelot.UnitTests/DependencyInjection/OcelotBuilderTests.cs

```diff
@@ -289,5 +351,18 @@ private void ThenAnExceptionIsntThrown()
         {
             _ex.ShouldBeNull();
         }
+
+        private class FakeCustomLoadBalancer : ILoadBalancer
+        {
+            public Task<Response<ServiceHostAndPort>> Lease(DownstreamContext context)
+            {
+                throw new NotImplementedException();
```

Comment:
Missing implementation or will be like this?

Reply:
Not relevant for ```OcelotBuilder``` tests.
I added a comment to the fake class.